ledger-app-fio/src/signTransaction.h

```diff
@@ -0,0 +1,34 @@
+#ifndef H_FIO_APP_SIGN_TRANSACTION
+#define H_FIO_APP_SIGN_TRANSACTION
+
+#include "handlers.h"
+#include "hash.h"
+#include "fio.h"
+
+handler_fn_t signTransaction_handleAPDU;
+
+typedef struct {
+	sha_256_context_t hashContext;
+	int ui_step;
+	int stage;
+
+	network_type_t network;
+	action_type_t action_type;
+	bip44_path_t path;
```

Comment:
The name is too general. If it is used for various purposes, it might be worth listing some examples here.